src/invidious/routes/embed.cr

```diff
@@ -12,13 +12,15 @@ module Invidious::Routes::Embed
           url = "/playlist?list=#{plid}"
           raise NotFoundException.new(translate(locale, "error_video_not_in_playlist", url))
         end
+
+        get_first_video = videos[0].as(PlaylistVideo)
```

Comment:
Do you mind renaming these to `first_video`?
`get_first_video` sound too much like a method or function.